packages/devtools_app/lib/src/screens/vm_developer/vm_service_private_extensions.dart

```diff
@@ -256,3 +256,8 @@ enum GuardClassKind {
     }
   }
 }
+
+/// An extension on [Library] which allows for access to VM internal fields.
+extension LibraryPrivateExtension on Library {
+  String? get vmName => json!['_vmName'];
```

Comment:
Nit: put `_vmName` into a private variable and use that as the key.

Reply:
Making `_vmNameKey` a top level constant